tajo-catalog/tajo-catalog-server/src/test/java/org/apache/tajo/catalog/TestCatalog.java

```diff
@@ -274,10 +274,8 @@ public void testDropDatabaseWithAllTables() throws Exception {
     Map<String, List<String>> createdTablesMap = createBaseDatabaseAndTables();
 
     // Each time we drop one database, check all databases and their tables.
-    Iterator<String> it = new ArrayList<>(createdTablesMap.keySet()).iterator();
-    while(it.hasNext()) {
+    for (String databaseName : new ArrayList<>(createdTablesMap.keySet())) {
```

Comment:
Oh, I found that my above comment has no relation about this issue.
Creation of this array is needed because of 284 line in same file.